tests: tests/drivers/spi/spi_loopback: mimxrt1180_evk/mimxrt1189/cm7: build failure

### Describe the bug

mimxrt1180_evk/mimxrt1189/cm7
tests/drivers/spi/spi_loopback build failure

git bisect find cc12a5ead60b2ae1bec936c1f8478b0a71f37a2e
```
commit cc12a5ead60b2ae1bec936c1f8478b0a71f37a2e
Author: Declan Snyder <declan.snyder@nxp.com>
Date:   Wed Jul 30 11:28:22 2025 -0500

    tests: spi_loopback: Add delay props to RT series

    RT series have LPSPI data input setup and hold requirements and CS lead/lag
    requirements not being met on the current configuration for this
    test, causing flaky data transfer on CPHA 0 and 2 tests.
    Fix by adding delay properties.

    Signed-off-by: Declan Snyder <declan.snyder@nxp.com>

```

### Regression

- [x] This is a regression.

### Steps to reproduce

scripts/twister -p mimxrt1180_evk/mimxrt1189/cm7 -j 12 -T tests/drivers/spi_loopback --build-only
or 
west build -b  mimxrt1180_evk/mimxrt1189/cm7 tests/drivers/spi_loopback

### Relevant log output

```shell
-- Board: mimxrt1180_evk, qualifiers: mimxrt1189/cm7-- Found host-tools: zephyr 0.17.1 (/opt/toolchains/zephyr-sdk-0.17.1-rc4)-- Found toolchain: zephyr 0.17.1 (/opt/toolchains/zephyr-sdk-0.17.1-rc4)-- Found Dtc: /opt/toolchains/zephyr-sdk-0.17.1-rc4/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") -- Found BOARD.dts: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm7.dts-- Found devicetree overlay: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/tests/drivers/spi/spi_loopback/boards/mimxrt1180_evk_mimxrt1189_cm7.overlaydevicetree error: 'transfer-delay' appears in /soc/memory@20484000/ocram@0 in /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1180_evk_mimxrt1189_cm7/zephyr/tests/drivers/spi/spi_loopback/drivers.spi.loopback/zephyr/zephyr.dts.pre, but is not declared in 'properties:' in /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/dts/bindings/base/zephyr,memory-region.yamlCMake Error at cmake/modules/dts.cmake:306 (execute_process): execute_process failed command indexes: 1: "Child return code: 1"Call Stack (most recent call first): cmake/modules/zephyr_default.cmake:131 (include) share/zephyr-package/cmake/ZephyrConfig.cmake:75 (include) share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate) cmake/package_helper.cmake:101 (find_package)
```

### Impact

Functional Limitation – Some features not working as expected, but system usable.

### Environment

- OS: ubuntu
- toolchain: 0.17.1
- sha: v4.2.0-1434-gba65ff34fbd4

### Additional Context

_No response_

--- a/tests/drivers/spi/spi_loopback/boards/mimxrt1180_evk_mimxrt1189_cm7.overlay
+++ b/tests/drivers/spi/spi_loopback/boards/mimxrt1180_evk_mimxrt1189_cm7.overlay
@@ -16,9 +16,6 @@
 
 &ocram1 {
 	zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM_NOCACHE) )>;
-	transfer-delay = <100>;
-	pcs-sck-delay = <100>;
-	sck-pcs-delay = <100>;
 };
 
 &lpspi3 {
@@ -32,4 +29,7 @@
 		reg = <0>;
 		spi-max-frequency = <16000000>;
 	};
+	transfer-delay = <100>;
+	pcs-sck-delay = <100>;
+	sck-pcs-delay = <100>;
 };

